### ISSUE TITLE
Some relaton entries have unexpected titles

### ISSUE THREAD

**Issue by ribose-jeffreylau** [CalConnect/cc-transcription-systems#86]
```asciidoc
[bibliography]
== Normative References

* [[[ISO3166,ISO 3166]]]

* [[[ISO639,ISO 639]]]
```

```diff
-ISO 3166 (all parts), Codes for the representation of names of countries and their subdivisions (<- expected)
+ISO 3166 (all parts), Codes for the representation of names of countries

-ISO 639 (all parts), Codes for the representation of names of languages (<- expected)
+ISO 639 (all parts), Code for the representation of names of languages
```

Is there any way around this?  (apart from patching the relaton cache manually?)

Source:

* [ISO 3166-1](https://www.iso.org/standard/72482.html)
* [ISO 639-1](https://www.iso.org/standard/22109.html)

**Comment by ronaldtse** [comment]
The reason why ISO 3166 has that title is that the last published "ISO 3166" is this one:
https://www.iso.org/standard/22748.html

<img width="1062" alt="Screenshot 2021-07-29 at 12 19 30 PM" src="https://user-images.githubusercontent.com/11865/127430917-fbcd4e4e-56fe-48e9-9e16-d5882b227e84.png">

The case with 639 is probably similar. In this case let's cite them individually.

This topic is a worthwhile FAQ question for Metanorma -- can you post this issue in metanorma/metanorma.org? Thanks!

P.S. Herein lies the perils of undated references...

**Comment by ribose-jeffreylau** [comment]
@ronaldtse Thanks. I'll post it in metanorma.

P.S. Found another mismatch with expectations:

```asciidoc
* [[[ISO8601,ISO 8601]]]
```

```diff
-ISO 8601 (all parts), Date and time — Representations for information interchange (<- expected)
+ISO 8601 (all parts), Data elements and interchange formats — Information interchange
```

**Comment by ronaldtse** [comment]
That's because the last "ISO 8601" (not ISO 8601-1) had that title. This is a discussion topic in itself -- if I cite "ISO 8601", what is actually cited? ISO 690 does not provide that answer. Maybe time to ping Juha (this is clearly a topic for a 690 support group!).

**Comment by ribose-jeffreylau** [comment]
Fixed by c9d4a3abca14ee9d3e0037e10b0c6af3027281da which is a workaround.  Now it seems only ISO 639 needs the patch.

**Comment by ronaldtse** [comment]
@ribose-jeffreylau instead of making a patch, can you file an issue at Relaton-ISO and propose a syntax so that we can cite it properly? Thanks.
